ReactNativeClient/lib/models/BaseItem.js

```diff
@@ -644,7 +644,7 @@ class BaseItem extends BaseModel {
 
 	static displayTitle(item) {
 		if (!item) return '';
-		return !!item.encryption_applied ? '🔑 ' + _('Encrypted') : item.title + '';
+		return !!item.encryption_applied ? '🔑 ' + _('Encrypted') : (item.title.length>0)? item.title + '' : Note.defaultTitle(item);
```

Comment:
Here you need to check that `item.title` is defined before checking the length. Probably just `!!item.title` should cover all cases (that title is defined and that its length is greater than 0).

Reply:
I will make this change and remove the unneeded tests.